src/Symfony/Component/Messenger/CHANGELOG.md

```diff
@@ -6,38 +6,34 @@ CHANGELOG
 
  * The component is not experimental anymore
  * All the changes below are BC BREAKS
+ * senders and handlers subscribing to parent interfaces now receive *all* matching messages, wildcard included
```

Comment:
New names for `route_message` and `call_message_handler` middleware aren't mentioned (respectively `send_message` and `handle_message`). I think it deserves an entry.

Reply:
I'm not sure it makes sense - the component doesn't use these names at all - FWB did, and they were mostly wiring things to me.

Reply:
That's true when you let the fwb define the default middleware stack. But you can actually disable this by setting `default_middleware` to `false` and explicitly define yours. So these names are used in userland.

Reply:
Oh sorry. Missed the point. Indeed that's not Messenger related, but Fwb related.